keps/sig-api-machinery/365-paginated-lists/README.MD

```diff
@@ -152,6 +248,86 @@ malicious users cannot trick the server into serving results outside of its rang
 The storage layer must perform defensive checking on the provided value, check for path traversal attacks, and have 
 stable versioning for the continue token.
 
+#### watchcache
+
+However, not all LIST requests are served from etcd. Our API allows clients to opt-in for the LIST result to be
+served from kube-apiserver cache (called watchcache), being propagated via watch from etcd3.
+However, up until release 1.24, support for pagination wasn't added to watchcache. This makes the logic that
+we use extremely non-intuitive, as it effectively works as following:
```

Comment:
@aojea we have talked about it recently, Wojtek explained the current semantics in a clear way, have a look